qemu/tests/numa_opts.py

```diff
@@ -1,49 +1,122 @@
 import logging
-from virttest.utils_misc import normalize_data_size
-
+import re
 
-logger = logging.getLogger(__name__)
-dbg = logger.debug
+from virttest import error_context
+from virttest import utils_package
+from virttest.utils_misc import normalize_data_size
+from virttest.utils_misc import get_mem_info
 
 
+@error_context.context_aware
 def run(test, params, env):
     """
     Simple test to check if NUMA options are being parsed properly
-
-    This _does not_ test if NUMA information is being properly exposed to the
-    guest.
```

Comment:
Will you please add test steps as other cases? Thanks.

Reply:
> Will you please add test steps as other cases? Thanks.

Sure, updated, thanks a lot!